.tekton/operator-bundle-pipeline.yaml

```diff
@@ -658,3 +661,92 @@ spec:
       - name: kind
         value: task
       resolver: bundles
+
+  - name: create-acs-style-snapshot
+    # Explicitly running after all other tasks to ensure that
+    # - there are no failures that should prevent a release of the operator-bundle image (missing RPMs signatures, deprecated base images, ...)
+    # - the source image is present as it is required by EC
+    runAfter:
+    - apply-tags
+    - build-source-image
+    - clair-scan
+    - clamav-scan
+    - deprecated-base-image-check
+    - rpms-signature-scan
+    - sast-snyk-check
+    - push-dockerfile
+    params:
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+    - name: IMAGE_REFS
+      value: |
+        [
+          {
+            "component": "central-db",
+            "ref": "$(params.central-db-image-build-repo)@$(tasks.wait-for-central-db-image.results.IMAGE_DIGEST)",
+            "repository": "$(params.git-url)",
+            "revision": "$(params.revision)"
+          },
+          {
+            "component": "collector",
+            "ref": "$(params.collector-full-image-build-repo)@$(tasks.wait-for-collector-full-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-collector-full-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-collector-full-image.results.GIT_REF)"
```

Comment:
[nit] There's a bit of naming clash between `"ref"` and `"revision"` that gets assigned to `...GIT_REF`. You could rename `"ref"` to `"image_ref"` to prevent that.

Have you considered creating this JSON in the format of Snapshot? I.e.

```json
            {
                "name": "...",
                "containerImage": "...",
                "source": {
                    "git": {
                        "revision": "...",
                        "url": "..."
                    }
                }
            },
```

Yes, it's a bit longer here but fewer processing in Python (only manipulate Container `name`).

Reply:
As discussed, renaming would be nice but changing structure isn't required.